test/admin/e2e/list-view/e2e.spec.ts

```diff
@@ -1283,6 +1302,39 @@ describe('List View', () => {
       await expect(page.locator('.row-2 .cell-number')).toHaveText('1')
     })
 
+    const openColumnControls = async () => {
```

Comment:
There is likely a helper function for this already in `test/helpers/e2e`, and if not, this should go there.